### DISCUSSION TITLE
질문이 있습니다.

### DISCUSSION THREAD

**Discussion by keikang** [tedilabs/fastcampus-devops#57] [Q&A] [answered]
Docker와 관련된 내용이 아닐수도 있는데
강의를 보다 보면 cd 로 경로 입력하시다가 단축키로 ls같이 하위 디렉토리를 볼 수 있는거 같은데
어떻게 검색해야 될 지를 몰라서 찾을수가 없습니다.
알려주시면 좋겠습니다~
강의 잘 듣고 있습니다!

**Comment by posquit0** [comment, marked as answer]
안녕하세요!
터미널 상에서 ls로 경로를 입력하시다가 Tab 을 누르시면 자동완성이 되거나, 자동완성 가능한 아이템이 2개 이상일 경우 해당 아이템들을 출력해주게 됩니다. ㅎㅎ

**Reply by keikang** [reply]
![화면캡춰](https://user-images.githubusercontent.com/59009362/159462668-b390a4a2-2b84-4421-8196-a221ee302b0f.jpg)
여기 캡춰 보시면 cd fastcampus-devops/ 하시고
엔터는 치지 않으신거 같은데 밑에 보면 하위 디렉토리가 나오는데
이걸 어떻게 하는건지 궁금합니다
캡춰 문제가 된다면 지울게요~
도저히 말로 설명할 방법이 없어서 올렸습니다...

**Reply by posquit0** [reply]
넵~ 질문 주신 내용 동일하게 이해하고 답변 드린 내용입니다! :)
키보드에서 tab 키를 누르시면 터미널에서 자동완성 기능이 동작하게 됩니다. 자동완성 후보 아이템이 여러개일 경우 위와 같이 하단에 리스팅 됩니다.

**Reply by keikang** [reply]
감사합니다! :)
